### PR TITLE
V0.3 qp workflow

### PR DESCRIPTION
**Pull request by pwalczysko**
In preparation of the Turku workshop, adding new features to the guide.

Ready for review.

cc @jburel

### CODE REVIEW

**Comment by pwalczysko** [comment]
Ready for review cc @melvingelbard

**Comment by melvingelbard** [comment]
Looks good to me!
Maybe it could directly link to the right section of the QuPath docs to refer back to it to avoid confusion (e.g.
https://qupath.readthedocs.io/en/latest/docs/advanced/omero.html#browsing-an-omero-server & https://qupath.readthedocs.io/en/latest/docs/advanced/omero.html#send-objects-back-to-your-omero-server).

I was a bit confused at the bit where it refers to `Detection`s as 'derived `Annotation`s'. `Annotation` and `Detection` objects are very similar in some ways but quite different in the way they are processed in QuPath. But I'm not sure this is very important in this case :)

It's very nice to see tutorials on this. Looking forward to working more on the extension and listening to more feedback for improvement!

File: docs/qupath.rst [hunk 1]
```diff
@@ -5,99 +5,141 @@ Description
...
 
-#. In the following steps, we will show how to convert the ROIs your just created in QuPath into OMERO ROIs and attach them to the image in OMERO.
+#. Go to OMERO.iviewer and refresh the image. Inspect the saved detection ROIs.
 
-#. First, use the ROI OME-XML export script to export your ROIs from QuPath into OME-XML file. Find the version of ``ome-omero-roitool`` mentioned in Resources on `ome-omero-roitool releases <https://github.com/glencoesoftware/ome-omero-roitool/releases>`_ and from there download the ``ome-omero-roitool-xxx.zip``. The downloaded zip contains both the plugin and the QuPath scripts needed for this workflow.
+.. _Roitool:
+
+Save detection ROIs using ome-omero-roitool
+-------------------------------------------
+This workflow necessritates the usage of the Command Line Interface. The limitation here are the Annotation ROIs, whcih are transformed into masks in OMERO. Although this preserves the holes in the Annotations, if the Annotation ROIs are too large, it might result in performance problems or even running out of resources on the machine where the export of the mask from QuPath is attempted.
```
Comment (will-moore) [inline docs/qupath.rst]: typo "necessritates"

Comment (will-moore) [reply docs/qupath.rst]: and "whcih"

File: docs/qupath.rst [hunk 2]
```diff
@@ -5,99 +5,141 @@ Description
...
+
+#. Connect QuPath to OMERO, open an image from OMERO in QuPath and draw an ``Annotation`` on it as described in :ref:`Opening images with ROIs from OMERO in QuPath<OpeninginQuPath>`.
+
+#. Adjust your ``Annotation`` using the ``Brush`` tool |image3|.
+
+#. Select ``Analyze > Cell detection > Cell detection``.
+
+#. You can adjust the parameters. Click ``Run``. This will draw red ROIs around cells and nuclei inside your ``Annotation``.
+
+#. Use the ROI OME-XML export script to export your ROIs from QuPath into OME-XML file. Find the version of ``ome-omero-roitool`` mentioned in Resources on `ome-omero-roitool releases <https://github.com/glencoesoftware/ome-omero-roitool/releases>`_ and from there download the ``ome-omero-roitool-xxx.zip``. The downloaded zip contains both the plugin and the QuPath scripts needed for this workflow.
```
Comment (will-moore) [inline docs/qupath.rst:142]: Now that you have this, I think you can remove the duplicated bits from steps below: "If you did not do so already, find the version of the ome-omero-roitool mentioned in Resources on ome-omero-roitool releases. From there, download the ome-omero-roitool-xxx.zip" and "Unzip the downloaded file" and even `unzip ome-omero-roitool-xxx.zip`.

**Comment by will-moore** [comment]
Looks good (at https://omero-guides--6.org.readthedocs.build/projects/qupath/en/6/qupath.html 👍 ).
Few minor comments. NB: I haven't tried running the steps myself.

**Comment by pwalczysko** [comment]
fixes pushed https://github.com/ome/omero-guide-qupath/pull/6/commits/e7fc3cc53dbe2bd0bdc047942bc7bb175888a4ea,  https://github.com/ome/omero-guide-qupath/pull/6/commits/7e49f727da0c8cba07b17156fd1f47812e8a10e0  cc @melvingelbard - Re the ``derived Annotations`` formulation - I have removed this formulation replacing it with ``derrived ROIs``. I have left at another place the ``Annotation ROIs`` though - let me know if this is acceptable.  The problem we are having here in terms of terminology is:

In OMERO, all ROIs are a subtype of annotaion, other annotations might be e.g. key-value pairs, or Tags etc.
In QuPath, afaik, the Annotations are what we would call in OMERO (for the lack of better terms) "special large ROIs". I would like to avoid this explanation in the workflow, as it is bound to be confusing.

**Comment by pwalczysko** [comment]
@will-moore fix pushed https://github.com/ome/omero-guide-qupath/pull/6/commits/00853dd5d6c8135e1f1e9258864952c96da462e1

File: docs/qupath.rst (hunk 2 above)
Comment (pwalczysko) [reply docs/qupath.rst:142]: fixed in https://github.com/ome/omero-guide-qupath/pull/6/commits/00853dd5d6c8135e1f1e9258864952c96da462e1

File: docs/qupath.rst (hunk 1 above)
Comment (pwalczysko) [reply docs/qupath.rst]: fixed in https://github.com/ome/omero-guide-qupath/pull/6/commits/00853dd5d6c8135e1f1e9258864952c96da462e1

**Comment by will-moore** [comment]
Looks great, thanks.

**Review by will-moore — APPROVED** [review]

**Comment by pwalczysko** [comment]
@will-moore thank you. @jburel can we merge please so that we are ready for next week's outreach ?

**Comment by jburel** [comment]
@pwalczysko I have adjusted the permissions on that repo, so that next time you can merge

**Comment by pwalczysko** [comment]
> @pwalczysko I have adjusted the permissions on that repo, so that next time you can merge

@jburel thank you

**Pull request merged**